packaging/arch/PKGBUILD

```diff
@@ -93,17 +120,20 @@ package_snapd() {
   # Install the snapd socket and service for the main daemon
   install -m 644 "$GOPATH/src/${_gourl}/data/systemd/snapd.service" "$pkgdir/usr/lib/systemd/system"
   install -m 644 "$GOPATH/src/${_gourl}/data/systemd/snapd.socket" "$pkgdir/usr/lib/systemd/system"
-  # Install snap and snapd executables
+  # Install snap, snapctl, snap-update-ns, snap-seccomp and snapd executables
```

Comment:
I would prefer not to list all binaries again in the comment as then we need to update it with every new added one and the comments gets bigger. Why not just "Install necessary exeutables for snapd"